src/main/scala/com/fulcrumgenomics/umi/ConsensusCaller.scala

```diff
@@ -119,7 +128,16 @@ class ConsensusCaller(errorRatePreLabeling:  PhredScore,
       * Returns the number of reads that contributed evidence to the consensus. The value is equal
       * to the number of times add() was called with non-ambiguous bases.
       */
-    def contributions: Int = this.contributors
+    def contributions: Int = this.observations.sum
+
+    /** Gets the number of observations of the base in question. */
+    def observations(base: Base): Int = base match {
+      case 'A' => this.observations(0)
+      case 'C' => this.observations(1)
+      case 'G' => this.observations(2)
+      case 'T' => this.observations(3)
+      case x   => throw new IllegalArgumentException("Unsupported base: " + x.toChar)
+    }
```

Comment:
Could also just return `this.observations` since folks have access to `DnaBases` above.  Perhaps both since this is a nice convenience method.